config/intellij-idea-inspections.xml

```diff
@@ -105,9 +105,11 @@
     <inspection_tool class="AssignmentUsedAsCondition" enabled="true" level="ERROR" enabled_by_default="true" />
     <!-- we are ok to use auto-boxing as we use modern java -->
     <inspection_tool class="AutoBoxing" enabled="false" level="ERROR" enabled_by_default="false" />
-    <!-- we can not use try-with-resource till we switch to jacoco for code coverage -->
-    <inspection_tool class="AutoCloseableResource" enabled="false" level="ERROR" enabled_by_default="false" />
-    <!-- we are ok to use auto-unboxing as we use modern java -->
+    <inspection_tool class="AutoCloseableResource" enabled="true" level="ERROR" enabled_by_default="true" >
+        <option name="ignoreFromMethodCall" value="true" />
+    </inspection_tool>
```

Comment:
comment for myself.
from IDEA:

>Reports AutoCloseable instances which are not used in a try-with-resources statement, also known as Automatic
 Resource Management. This means that the open resource before/in try, close in finally style which was used 
before try-with-resources was available is also reported. This inspection is meant to replace all opened but not 
safely closed inspections when developing in Java 7 and higher.
Use the first table below to specify which AutoCloseable subclasses should be ignored by this inspection. Specify
AutoCloseable subclasses here which do not need to be closed.
Use the second table below to specify which methods returning AutoCloseable will be ignored when called.
Use the first checkbox below to ignore an AutoCloseable if it the result of a method call. When enabled, the results 
of factory methods will also be ignored.
Use the second checkbox below to specify that the inspection should not warn if an AutoCloseable instance is 
passed as a method call argument. If enabled the inspection assumes the resource is closed in the called method. Method calls inside a finally block with close in the name and an AutoCloseable argument will not be ignored.

ignoreFromMethodCall is true by default.

Reply:
Yes, it is related to the `ignoreFromMethodCall` option.
By turning it off the IDEA reports the line
```
            out = Files.newOutputStream(Paths.get(outputLocation));
```
as violation in the same way as Eclipse does. One more: the suppression
```
    @SuppressWarnings("resource")
```
Works for IDEA in the same way as for Eclipse. So it is possible to set `ignoreFromMethodCall` to false.